src/types/Options.ts

```diff
@@ -1,9 +1,15 @@
 import { DefaultEnvVars, UndefinedEnvVars } from './EnvVars';
 import { RemoveKeys } from './helpers';
 
+export interface ConfigOptions {
+  envFile?: string;
+  loadDotEnv?: boolean;
```

Comment:
What's the value of the `envFile` without the flag to load it? Why, as a user, would I ever use an `envFile` and not load it in?

@johnkahn I also think `envPath` might be clearer... thoughts?

Reply:
I agree on both points, I'll update accordingly